ext/pcntl/tests/001.phpt

```diff
@@ -25,13 +24,11 @@ function test_exit_signal(){
 	print "\n\nTesting pcntl_wifsignaled....";
 
 	$pid=pcntl_fork();
-    
+
 	if ($pid==0) {
-		sleep(10);
-		exit;
+		posix_kill(posix_getpid(), SIGTERM);
 	} else {
 		$options=0;
-		posix_kill($pid, SIGTERM);
```

Comment:
I am able to reproduce the travis failure locally on a RHEL 7.1.1503 box too, hence the commit. 

In the strace I pasted in the commit message, you can see the tgkill being sent to the wrong thread-group-id. PID 111758 is the forked child and 111740 is the parent.
```    
[pid 111758] 1467239169.831934 tgkill(111740, 111758, SIGTERM) = -1 ESRCH (No such process) <0.000012>
```
The race condition is that after the fork, the child process isn't finished being set up (including the call to set_robust_list()) but the master process was already scheduled and sent the kill.